lib/internal/util/debuglog.js

```diff
@@ -31,7 +31,7 @@ function emitWarningIfNeeded(set) {
   }
 }
 
-function debuglog(set) {
+function debuglogImpl(set) {
   set = set.toUpperCase();
   if (!debugs[set]) {
     if (debugEnvRegex.test(set)) {
```

Comment:
Note that since `process.env.NODE_DEBUG` is saved above during pre-execution, so even though the saved value is accessed lazily, it's still saved as soon as possible.